### PR TITLE
Update the sample

### PR DESCRIPTION
**Pull request by andreysaf**
- Added open/save buttons
- reformatted from IIF to render.js

### CODE REVIEW

**Comment by andreysaf** [comment]
@achenPDFTron could you please verify that everything runs on Windows?

File: src/render.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  instance.setTheme("dark");
+  instance.disableElements(['downloadButton']);
+
+  const { docViewer, annotManager } = instance;
+
+  openFileBtn.onclick = async () => {
+    const file = await dialog.showOpenDialog({
+      properties: ["openFile", "multiSelections"],
+      filters: [
+        { name: "Documents", extentions: ["pdf", "docx", "pptx", "xlsx"] },
```
Comment (achenPDFTron) [inline src/render.js]: `extentions`

is that a typo?

Comment (andreysaf) [reply src/render.js]: Fixed the typo. Thanks Anthony, does it run on Windows fine?

Comment (achenPDFTron) [reply src/render.js]: did some quick smoke tests on my end by quickly adding some annotations and trying out printing/downloading.

things looked fine.

**Review by achenPDFTron — APPROVED** [review]

**Pull request merged**